### ISSUE TITLE
Support CREATE INDEX

### ISSUE THREAD

**Issue by adonespitogo** [joe-re/sql-language-server#185]
Hi @joe-re 

In addition to supporting [CREATE TABLE](https://github.com/joe-re/sql-language-server/issues/129), can I also ask to implement `CREATE INDEX` feature? Here's an example of a basic create index statement I'm using in my migration:

```mysql
CREATE INDEX IF NOT EXISTS index_mac_address ON devices(mac_address);
```
As much as I want to contribute to your project, I just don't have the luxury of time for now to dig into the source codes. Although this is not a priority, it would be nice to see all diagnostic errors disappear in my neovim file explorer `nvim-tree`.

**Comment by joe-re** [comment]
@adonespitogo HI, thanks for your report!
Even if you don't have enough time to make a PR, just reporting the problem is great for us to improve it. 🙏 

Sorry I'm not familiar with nvim-tree but can you elaborate more about what you mean by "see all diagnostic errors disappear"?
Probably nvim-tree can manage multiple files opening state and you want to see errors for all of them?

**Comment by adonespitogo** [comment]
Hi @joe-re 

`nvim-tree` is a file explorer plugin for `neovim`. It shows the files that are having diagnostic/syntax errors. These errors come from the sql files which have `CREATE INDEX`  and `DROP TABLE` statements that are not yet supported by `sqlls`:

![Screenshot from 2023-04-18 10-51-02](https://user-images.githubusercontent.com/3839684/232658970-e88d5ad9-bbf1-4398-ad22-f701d324ec3a.png)

And example of diagnostic error in one of my files:

![Screenshot from 2023-04-18 10-50-07](https://user-images.githubusercontent.com/3839684/232659135-c0a83620-e5e4-49c7-b7a6-482c9ad17ca9.png)

**Comment by joe-re** [comment]
@adonespitogo We released v1.5.1 and not it supports basic statement of create index.
Please check it out and if you find any problems please let us know. Thanks.
